internals/cli/cli.go

```diff
@@ -363,7 +363,7 @@ func errorToMessage(e error) (normalMessage string, err error) {
 	return msg, nil
 }
 
-func getEnvPaths() (pebbleDir string, socketPath string) {
+func getEnvPaths() (pebbleDir string, socketPath string, pebbleImports []string) {
```

Comment:
I realise this is an internal function, but maybe it's time to have this return a little `type envPaths struct { ... }` with named fields, rather than relying on the order of return arguments.

Reply:
sounds good.